tests/test_actions.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for the `edx-enterprise` admin actions module.
+"""
+from __future__ import absolute_import, unicode_literals
+
+import unittest
+
+import factory
+import mock
+
+from enterprise.actions import export_as_csv_action
+
+
+class DummyModel(object):
```

Comment:
@clintonb For two reasons:

1. To emphasize that the action works with any model
2. `test_export_as_csv_callable_fields` needs a callable attribute on a model, while `EnterpriseCustomer` does not have any properties/methods so far, and I don't want to create one on the actual model just for the test purposes.